src/commands.ts

```diff
@@ -89,6 +102,32 @@ export function addCommands(
       await settings.set('starredSection', !starredSection);
     }
   });
+  app.commands.addCommand(CommandIDs.showNotebookLauncher, {
+    isToggleable: true,
+    isToggled: () => {
+      return settings.composite
+        .launchNotebookSection as ISettingsLayout['launchNotebookSection'];
+    },
+    label: trans.__('Show create new empty notebook section'),
```

Comment:
```suggestion
    label: trans.__('Show "Create Notebook" section'),
```

This was an odd one:

![image](https://github.com/user-attachments/assets/f1e03356-18e7-49a5-ba06-c9024a90d471)